modules/ROOT/pages/design-trusted-domains.adoc

```diff
@@ -0,0 +1,20 @@
+= Allowlisting Trusted Domains
+
+Every HTTP request you make to an external resource goes through the API Console Proxy. For security reasons, you need to register the domain as a trusted domain in Anypoint Platform.
+
+To add a new domain:
+
+1. Go to Access Management.
+2. Click *Trusted domains*. 
+3. Click *Add domain*. +
+The domain must not match with any existing trusted domain, and must not be any domain or subdomain of Mulesoft or Anypoint Platform.
+4. In the dialog box that appears, type the domain name and click *Add domain*. +
+_Result_: The domain has been added to your list of trusted domains. You are redirected to the trusted domains page, where you can edit or delete your domains.
+
+== Automatic Registration of Domains
+
+When publishing an API specification to Exchange, or sending a request trough the proxy, the base URI of the implementation will automatically be added to the list of trusted domains. Administrators will receive an email notifying you of this event. The email will redirect you to the *Trusted domains* page in Access Management, where you can add or delete new domains as explained before.
+
+NOTE: The allowlisting of trusted domains will be rolled in two phases. During the first phase, non-authenticated request will be automatically registered. During the second phase, non-authenticated requests won't be automatically added to the list of trusted domains and will fail if the domain is not manually added to the list.
```

Comment:
I think we should be clearer on the dates, specifically when talking about the second phase, otherwise, customers will be unsure of when the enforcement will be applied. Let me work with our PM on this

Reply:
It's ok to not add specific dates here, we will run an analysis at the 30 day mark to see the feature adoption and based on that we should send a targeted email to org admins or add an in application message on the second phase date.

Reply:
Additionally, is there another way to address a non-authenticated user to be more specific that any request from the public portal or via shared link of mocking service will not be registered? or is this already somewhere in our docs?